### ISSUE TITLE
using r2l over the network fails

### ISSUE THREAD

**Issue by AmeBel** [opencog/opencog#1987]
### Steps taken
```scheme
cd /opencog/build
sudo make install                   
cd ../opencog/nlp/relex2logic/      
guile -l run-r2l.scm                
scheme@(guile-user)> (define relex-server-host "172.17.0.2")                           
scheme@(guile-user)> (nlp-parse "this is a test")                                      
ERROR: In procedure connect:                                                           
ERROR: In procedure connect: Connection refused                                        
                                                                                       
```
Previously (before the modulearization) setting `relex-server-host` sufficed when running from the cogserver.

**Comment by linas** [comment]
Is the relex server running? `Connection refused` is a standard network error, usually indicating that the remote end is not there, is blocked by firewall rules, the netowrk is disconnected, etc.

**Comment by AmeBel** [comment]
Yes it is. I am able to access it through `telnet 172.17.0.2 4444`

**Comment by leungmanhin** [comment]
I think it's because guile is only reading the IP that was installed (via `sudo make install`)

**Comment by AmeBel** [comment]
thanks manhin. Iit was the modulriazation, private vs public thing. Fixed by #1989
